pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -222,7 +222,10 @@ def on_run_end(self) -> List[List[STEP_OUTPUT]]:
 
     def teardown(self) -> None:
         """Frees memory of tracked epoch outputs."""
-        self.epoch_output = None
+        self._epoch_output = None
```

Comment:
This was a bug, we use `_` everywhere else

Reply:
thanks